pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```diff
@@ -15,15 +15,19 @@ spec:
     type: string
     description: "[S3 ONLY] S3 bucket name where the model is stored"
     default: ""
-  - name: gitServer
+  - name: containerfileGitServer
     type: string
-    description: Root domain of the Git server
-  - name: gitOrgName
+    description: Root domain of the Git server where the Containerfile repository is located
+  - name: containerfileGitOrgName
     type: string
-    description: Organization or Group name where the repo is located
-  - name: gitRepoName
+    description: Organization or Group name where the Containerfile repository is located
+  - name: containerfileGitRepoName
```

Comment:
This might be a good opportunity to turn these into a single parameter, similar to how `git-model-repo` is a single parameter, WDYT?

Are there any cases where we use these parts independently, or are we always constructing a single string composed of all 3 in the same order all the time?

Reply:
@grdryn actually we don't use them individually, I was thinking whether the whole or partial approach is better but it is quite common to input the URL as a whole, so I will change that. Thanks for input.

Reply:
Done.